### DISCUSSION TITLE
"Start to system tray" issue

### DISCUSSION THREAD

**Discussion by davidovic999** [mikeberger/borg_calendar#204] [General]
Hello Mike,
I activated the "Start to system tray" function.
Because I thought it was an autostart function. Borg doesn't start automatically.
In the system tray, there's just the Borg icon. And that's all!

![start sys tray](https://github.com/mikeberger/borg_calendar/assets/59475468/052d1e87-c8ac-4513-8565-9c9377da91fe)

Now, Borg's interface or dashboard no longer appears on the screen!

I closed the program using the "Task Manager" and restarted it. I also restarted Windows. But the situation is the same: no Borg interface on the screen!  

What's more, it's impossible to right- or left-click on the system tray icon! No options are displayed.

**Comment by mikeberger** [comment]
The start to system tray works fine for me on windows 11. I can left or right click on the tray icon to bring up a menu.

It is concerning though. I had to switch to use a completely different system tray icon library when the prior one stopped working on linux. The new system tray was actually the reason for the last release.

You can check .borg.log and .borg.out in your user folder (%USERPROFILE%) to see if there were errors.

**Comment by davidovic999** [comment]
I have Windows 10. The "System tray" icon doesn't work for me since I installed Borg. And even when I reinstalled it, still doesn't work!
The icon is there, but I can't right- or left-click to get the program options!

Now, as I explained, the Borg interface no longer appeared when I activated the "Start to system tray" option.

How can I make this interface reappear?

Here's the link to the 'log' and 'out' files : [https://ufile.io/5xyrlver](url)

**Comment by mikeberger** [comment]
Just to get you going, you can delete registry entry HKEY_CURRENT_USER\Software\JavaSoft\Prefs\net\sf\borg\common\util\backgstart to shut off "start to system tray".

You can edit HKEY_CURRENT_USER\Software\JavaSoft\Prefs\net\sf\borg\common\util\use/Sys/Tray and change it to false to shut off the system tray icon completely.

There were no errors in the log files.

**Comment by mikeberger** [comment]
It looks like other people have encountered the same problem with this third-party system tray: https://github.com/dorkbox/SystemTray/issues/209

**Comment by mikeberger** [comment]
I built a version that uses the old system tray for windows. You can download it here:
https://sourceforge.net/projects/borg-calendar/files/drops/BorgCalendar-1.10.04-3.msi/download

It also has some of the UI changes you suggested.

**Comment by davidovic999** [comment]
I followed your instructions. It works.
Thank you for your program UI updates.
